src/vcpkg-test/manifests.cpp

```diff
@@ -1045,11 +1045,34 @@ TEST_CASE ("license serialization", "[manifests][license]")
     CHECK(test_serialized_license("MIT") == "MIT");
     CHECK(test_serialized_license("mit") == "MIT");
     CHECK(test_serialized_license("MiT    AND (aPACHe-2.0 \tOR   \n gpl-2.0+)") == "MIT AND (Apache-2.0 OR GPL-2.0+)");
+    CHECK(test_serialized_license("uNkNoWnLiCeNsE") == "uNkNoWnLiCeNsE");
 }
 
 TEST_CASE ("license error messages", "[manifests][license]")
 {
     Parse::ParseMessages messages;
+    parse_spdx_license_expression("", messages);
+    REQUIRE(messages.error);
+    CHECK(messages.error->format() == R"(<license string>:1:1: error: SPDX license expression was empty.
```

Comment:
If we are checking the contents of localized strings in tests we need to ensure that vcpkg-test ignores localization. This might already be the case, but I don't recall having discussed it.

Reply:
That seems like a separable bugfix level change to me.

Reply:
vcpkg-test does ignore localization.